tasks/cdn-cli.js

```diff
@@ -0,0 +1,188 @@
+import chalk from 'chalk';
+import textTable from 'text-table';
+import { toCdnEntryName } from './cdn-entry-name.js';
+import { CDN_ENVIRONMENTS } from './cdn-environments.js';
+import { CdnManager } from './cdn-manager.js';
+import { NO_CACHE } from './cellar-client.js';
+import { getCurrentBranch } from './git-utils.js';
+
+async function run () {
+  const args = getArgs();
+
+  if (args.command === '-h' || args.command === 'help') {
+    return help();
+  }
+
+  const cdnEnvironment = getCdnEnvironment(args);
+  switch (args.command) {
+    case 'get':
+      return get(cdnEnvironment, await getCdnEntryName(cdnEnvironment, args));
+    case 'list':
+      return list(cdnEnvironment);
+    case 'publish':
+      return publish(cdnEnvironment, await getCdnEntryName(cdnEnvironment, args));
+    case 'delete':
+      return remove(cdnEnvironment, await getCdnEntryName(cdnEnvironment, args));
+    case 'ui':
+      return publishUi(cdnEnvironment);
+  }
+
+  console.log(chalk.red.bold('[!] Unknown command'));
+  printAvailableCommands();
+}
+
+function getArgs () {
+  const [command, env, name] = process.argv.slice(2);
+  return {
+    command,
+    env,
+    name,
+  };
+}
+
+/**
+ * @param {CdnEnvironment} cdnEnvironment
+ * @param {Object} args
+ * @return {Promise<string>}
+ */
+async function getCdnEntryName (cdnEnvironment, args) {
+  const name = cdnEnvironment.semver ? args.name : args.name ?? await getCurrentBranch();
+  return toCdnEntryName(cdnEnvironment, name);
+}
+
+/**
+ * @param {Object} args
+ * @return {CdnEnvironment}
+ */
+function getCdnEnvironment (args) {
```

Comment:
suggestion: maybe we should only pass `env` instead of `args` and reading `args.env`